packages/lodestar/src/chain/interface.ts

```diff
@@ -40,6 +40,7 @@ export interface IBeaconChain {
   readonly executionEngine: IExecutionEngine;
   // Expose config for convenience in modularized functions
   readonly config: IBeaconConfig;
+  readonly anchorSlot: Slot;
```

Comment:
would be nice to have a js doc:

Something like:
The initial slot that the chain is started with.